docs/reference.html

```diff
@@ -19248,7 +19248,7 @@ <h3 id='latlng-example'>Usage example</h3>
 <p>All Leaflet methods that accept LatLng objects also accept them in a simple Array form and simple object form (unless noted otherwise), so these lines are equivalent:</p>
 <pre><code>map.panTo([50, 30]);
 map.panTo({lng: 30, lat: 50});
-map.panTo({lat: 50, lng: 30});
+map.panTo({lat: 50, lng: 30}); // Note: `lon` can be used interchangably with `lng` in objects.
```

Comment:
Please revert this file, it is generated over a script. It's enough to change the doc in `LatLng.js`